development.postgresql.yml

```diff
@@ -0,0 +1,72 @@
+version: "2"
+services:
+  server:
+    build:
+      context: .
+      dockerfile: server.Dockerfile
+    networks:
+      serverdb:
+      signer:
+      server:
+        aliases:
+          - notary-server
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-server -config=fixtures/server-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/server/postgresql
+      DB_URL: postgres://server@serverdb:5432/notaryserver?sslmode=disable
+    depends_on:
+      - serverdb
+      - signer
+  signer:
+    build:
+      context: .
+      dockerfile: signer.Dockerfile
+    networks:
+      signerdb:
+      signer:
+        aliases:
+          - notarysigner
+    entrypoint: /usr/bin/env sh
+    command: -c "./migrations/migrate.sh && notary-signer -config=fixtures/signer-config.postgres.json"
+    environment:
+      MIGRATIONS_PATH: migrations/signer/postgresql
+      DB_URL: postgres://signer@signerdb:5432/notarysigner?sslmode=disable
+    depends_on:
+      - signerdb
+  serverdb:
```

Comment:
What we want as maintainers is a consistent way to do things, and in this instance, you have created something that differs from the most similar existing example in the repo, mariadb. 

You should have no reason to create a `notarypostgres` image, and we would ask you to change it if you did. We deprecated `notarymysql` because there was no need for us to use a custom built image when we could simply use the official images and their built in bootstrapping, speeding up our build and test times. Postgres has this same built in boostrapping as mariadb, you can find the instructions under ["How to extend this image"](https://hub.docker.com/_/postgres/), and an example of our basic setup for mariadb, which should be very similar, [here](https://github.com/docker/notary/tree/master/notarymysql/docker-entrypoint-initdb.d).